pennylane/qchem/structure.py

```diff
@@ -310,6 +341,15 @@ def hf_state(electrons, orbitals):
     >>> state = hf_state(2, 6)
     >>> print(state)
     [1 1 0 0 0 0]
+
+    >>> state = hf_state(2, 6, basis="parity")
+    >>> print(state)
+    [1 0 0 0 0 0]
```

Comment:
It currently gives: `[1. 0. 0. 0. 0. 0.]`, should be an easy fix by casting to int at some point.